docs/_data/table/en/ReadType.yml

```diff
@@ -4,6 +4,6 @@ CACHE_PROMOTE:
   for each completely read data block. This is the default read type.
 CACHE:
   If the data was not in the Alluxio storage of the local worker, a replica will be added to the
-  local Alluxio worker for each completely read data block.
```

Comment:
@maobaolong Thanks for updating

Could you also mention that when partial reads are set to false, the block must be read completely for it to be cached?